vllm/executor/multiproc_hpu_executor.py

```diff
@@ -0,0 +1,51 @@
+from typing import Callable, Optional, Tuple, Type
+
+import habana_frameworks.torch  # noqa: F401
+import torch
+
+from vllm.executor.multiproc_gpu_executor import (
+    MultiprocessingGPUExecutor, MultiprocessingGPUExecutorAsync)
+from vllm.logger import init_logger
+from vllm.utils import make_async
+from vllm.worker.worker_base import WorkerBase
+
+logger = init_logger(__name__)
+
+
+class MultiprocessingHPUExecutor(MultiprocessingGPUExecutor):
```

Comment:
I dont see how it does initialize hpu executor, _get_worker_module_and_class initializes proper worker but dont we need hpu executor also?

Reply:
so far we don't need anything from hpu executor, we're inheriting the entire logic from MP gpu executor